app/lib/methods/sendFileMessage.ts

```diff
@@ -37,9 +37,9 @@ export async function cancelUpload(item: TUploadModel): Promise<void> {
 export function sendFileMessage(
 	rid: string,
 	fileInfo: IUpload,
-	tmid: string,
+	tmid: string | undefined,
```

Comment:
```suggestion
	tmid?: string,
```

Reply:
The function params are messed up. If I mark as optional, it's going to complain about the following params not being optional as well.
Assign as `string | undefined` does the trick without any complaints.